opensciencegrid/mailchimp-sync/Dockerfile

```
@@ -0,0 +1,16 @@
+ARG BASE_YUM_REPO=release
+ARG BASE_OSG_SERIES=3.6
+
+FROM opensciencegrid/software-base:$BASE_OSG_SERIES-el8-$BASE_YUM_REPO
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+RUN yum -y install \
+             vim && \
+    yum -y clean all
```

```suggestion
FROM almalinux:8

LABEL maintainer OSG Software <help@osg-htc.org>

RUN yum -y install python3 python3-requests && \
    yum -y clean all
```
I see nothing OSG-specific here that would benefit from using the software-base image so just use a stock EL8.  (Suggesting alma because of https://opensciencegrid.atlassian.net/browse/SOFTWARE-5178.)